src/Exception/RouterException.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+namespace TrendingRepos\Exception;
+
+class RouterException extends \RuntimeException {}
```

Comment:
According to coding standards, an empty class should be formatted this way:

```suggestion
class RouterException extends \RuntimeException
{
}
```